scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1201,7 +1201,7 @@
                sort-jobs-duration
                (->> tasks
                     (group-by util/task-ent->user)
-                    (pc/map-vals (fn [task-ents] (into (sorted-set-by task-comparator) task-ents)))
+                    (pc/map-vals (fn [task-ents] (sort task-comparator task-ents)))
```

Comment:
Do we need the `set` behavior, i.e. is it possible for there to be duplicates in `task-ents`?
If there can be duplicates, maybe we first create a set and then do the sort?